packages/auth/ios/RNFBAuth/RNFBAuthModule.m

```diff
@@ -1667,7 +1667,8 @@ - (NSDictionary *)firebaseUserToDict:(FIRUser *)user {
       @"uid" : hint.UID,
       @"factorId" : hint.factorID == nil ? [NSNull null] : [self getJSFactorId:(hint.factorID)],
       @"displayName" : hint.displayName == nil ? [NSNull null] : hint.displayName,
-      @"enrollmentDate" : enrollmentDate,
+      @"enrollmentTime" : enrollmentDate,
```

Comment:
This is a breaking change. We might want to keep `enrollmentDate` around for a bit for backwards-compatibility in case people have worked around this discrepancy between platforms based on the OS